examples/v2.5.3/rx_messages

```diff
@@ -88,7 +88,7 @@
 # Modbus RTU Messages
 # ------------------------------------------------------------
 # [Address ][ Function Code] [ Data ][ CRC ]
-#  1b         1b               Nb      2b   
+#  1b         1b               Nb      2b
```

Comment:
these changes worries me, I cannot get pylint/black/flake8 to report them (with our CI setup).

Have you managed to get an error (not in the pre-commit hook) ?

Reply:
Strange.  `black` does remove the trailing whitespace, but only if you feed it a file directly.
```
❯ black examples/
All done! ✨ 🍰 ✨
44 files left unchanged.

❯ black examples/v2.5.3/rx_messages
reformatted examples/v2.5.3/rx_messages

All done! ✨ 🍰 ✨
1 file reformatted.
```

Reply:
you can configure black in the yaml file, like e.g.:
```
- repo: https://github.com/psf/black
    rev: 23.1.0
    hooks:
      - id: black
        args:
          - --quiet
        files: ^((homeassistant|pylint|script|tests)/.+)?[^/]+\.py$
```
and use the same arguments as CI uses.